src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double topBase;
+    private double bottomBase;
+    private double leg;
+    private double baseAngle;
+
+    public IsoscelesTrapezoid(String color, double topBase, double bottomBase,
+                              double leg, double baseAngle) {
+        super(color);
+        this.topBase = topBase;
+        this.bottomBase = bottomBase;
+        this.leg = leg;
+        this.baseAngle = baseAngle;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (topBase + bottomBase) / 2 * leg * Math.cos(90.0 - baseAngle);
```

Comment:
All magic numbers in your code should be constants. The number 90.0 seems to be a magic number, consider defining it as a constant field.